addons/l10n_ro_edi/models/account_edi_xml_ubl_ciusro.py

```diff
@@ -76,7 +76,7 @@ def _get_document_type_code_vals(self, invoice, invoice_data):
         # EXTENDS 'account_edi_ubl_cii
         # http://www.datypic.com/sc/ubl20/e-cbc_DocumentTypeCode.html
         vals = super()._get_document_type_code_vals(invoice, invoice_data)
-        vals['value'] = "380" if invoice.move_type == 'out_invoice' else "381"
+        vals['value'] = '50' if invoice.move_type == 'out_refund' else False
```

Comment:
Reading this:
```
((cbc:DocumentTypeCode='130') or
  ((local-name(/*) = 'CreditNote') and (cbc:DocumentTypeCode='50')) or
  (not(cbc:ID/@scheme) and not(cbc:DocumentTypeCode)))
  ```
It feels that not having `130` will get the document rejected too.
Shouldn't we put something like this to always have the value and fallback on false in case it would be an entry? (which shouldn't happen but just in case)
```suggestion
        vals['value'] = { 'out_invoice': '130', 'out_refund'}.get(invoice.move_type, False)
```

Reply:
I thought about this too but then I got an error from this rule (after putting '130' on an out_invoice move):
```
      <assert id="UBL-CR-666" flag="fatal" test="not(//cac:AdditionalDocumentReference[cbc:DocumentTypeCode = '130']/cac:Attachment)">[UBL-CR-666]-A UBL invoice shall not include an AdditionalDocumentReference simultaneously referring an Invoice Object Identifier and an Attachment</assert>
```
And we need the 'Attachment' for the pdf.
I have tested the current solution on the official validator website and it works to just discard the DocumentTypeCode for the out_invoice :)

Reply:
Fine by me! :ok_hand: